src/pages/ProductLists/ProductLists.js

```diff
@@ -1,10 +1,98 @@
 import React from 'react';
 import { withRouter } from 'react-router';
+import { API } from './config.js';
+import SearchFilter from './Components/SearchFilter';
+import ProductCard from './Components/ProductCard';
+import Dropdown from './Components/Dropdown';
 import './ProductLists.scss';
 
 class ProductLists extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      productBox: [],
+      value: '전체보기',
+    };
+  }
+
+  componentDidMount() {
+    fetch(`${API.PRODUCT}`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({
+          productBox: data.response,
+        });
+      });
+  }
+
+  handleFilter = e => {
+    this.setState({ value: e.target.value });
+    //전체보기
+    fetch(`${API.PRODUCT}`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ productBox: data.response });
+      });
+    //신상품순
+    fetch(`${API.PRODUCT}?sort=manufacture_date`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ productBox: data.response });
+      });
+    //인기순
+    fetch(`${API.PRODUCT}?sort=stock`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ productBox: data.response });
+      });
+    //낮은 가격순
+    fetch(`${API.PRODUCT}?sort=-price`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ productBox: data.response });
+      });
+    //높은 가격순
+    fetch(`${API.PRODUCT}?sort=price`)
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ productBox: data.response });
+      });
+  };
```

Comment:
- 현재는 필터를 누를 때마다 데이터에 대한 fetch 요청을 5번 보냅니다. 실제로 우리가 원하는 동작은 선택한 filter 에 대한 데이터 요청 1번만 보내는 것 이므로, 다음과 같이 구성해볼 수 있을 것 같습니다!
```suggestion
  handleFilter = e => {
    const { value } = e.target;
    fetch(`${API.PRODUCT}?sort=${value}`)
      .then(response => response.json())
      .then(data => {
        this.setState({ productBox: data.response });
      });
  }
```
- 이렇게 하시려면, DropDown 에서 option 태그의 value 부분에 각 필터에 해당하는 value 값이 들어가야 할 것 같습니다.
```jsx
<option value='-price'>낮은 가격순</option>
```